app/src/main/res/values/strings.xml

```diff
@@ -528,10 +528,10 @@
     <string name="enable_disposed_exceptions_summary">Force reporting of undeliverable Rx exceptions outside of fragment or activity lifecycle after disposal</string>
     <string name="show_original_time_ago_title">Show original time ago on items</string>
     <string name="show_original_time_ago_summary">Original texts from services will be visible in stream items</string>
-    <string name="disable_media_tunneling_title">Disable media tunneling</string>
```

Comment:
Same here with a key being called "disable" not containing disable. Same for other strings that were adjusted.

Also what is the reasoning behind changing this? I find that "disable" is a better word than "no"

Reply:
I presume media tunneling isn't still on.
Having the option say "No" when it could have been "Media tunneling" defaulted to "On",
makes it clearer that turning it off is only done actively, and that the default is "On".